tests/server/query_params/test_sort.py

```diff
@@ -1,30 +1,15 @@
-from datetime import datetime, timezone
-
 import pytest
 
 from optimade.server.warnings import FieldValueNotRecognized
+from optimade.server.data import structures
```

Comment:
Change the `structures` pytest fixture instead of importing this (in this way you can avoid the awkward sorting in `optimade/server/data/__init__.py` as well).

Reply:
The sorting is to do with database insertion, not these tests, but I guess this can still be parceled into a fixture (I don't think follow your issue with importing things outside the tests though)

Reply:
I think it mainly concerns loading the server-configuration. There might be situations where the config is loaded outside the test and take precedence. At least I remember we had issues with this previously. If you don't see any issues here, fine.